src/CustomerioTracking.tsx

```diff
@@ -181,8 +181,11 @@ class CustomerIO {
    * Call this method when a user interacts and taps open the push notification.
    * @param payload Customer.io payload as received from the push notification
    */
-  static async trackNotificationResponseReceived(payload: Object) {
-    if (payload == null) {
+  static trackNotificationResponseReceived(payload: Object) {
+    // Tracking push notification metrics on Android is handled automatically
+    // through the Google Services API, so there is no need to make a specific call for it.
+    // This method is specific to iOS and works as expected on Android without any additional intervention.
+    if (payload == null || Platform.OS === 'android') {
```

Comment:
I wonder if it would be valuable to make a re-usable function for this? It could get annoying to copy/paste this code over and over. 

```suggestion
    if (payload == null || isAndroid()) {
```

Reply:
Great suggestion. Implemented.